diktat-analysis.yml

```diff
@@ -10,7 +10,8 @@
   configuration: {}
 - name: ENUM_VALUE
   enabled: true
-  configuration: {}
+  configuration:
+    enumStyle: snakeCase
```

Comment:
I am afraid to ask, but may be we need to support all the cases?

Reply:
I know only about usage of these two :)

Reply:
yeah, agreee